build/linux/installer/conf/fluent-bit-network-flow-logs.conf

```diff
@@ -0,0 +1,31 @@
+[INPUT]
+    Name                tail
+    Alias               oms_networkflow_input
+    Tag                 oms.networkflow.*
+    Path                /var/log/acns/hubble/events.log
+    Read_from_Head      true
+    DB                  /var/opt/microsoft/docker-cimprov/state/network_flow.db
+    DB.Sync             Off
+    Parser              json
+    Rotate_Wait         20
+    Refresh_Interval    5
+    storage.type        filesystem
```

Comment:
1. instead of storage.type        filesystem and use  #${AZMON_STORAGE_TYPE}
2. add   #${AZMON_TAIL_THREADED}
 Final config would be something like


[INPUT]
    Name                tail
    Alias               oms_networkflow_input
    Tag                 oms.networkflow.*
    Path                /var/log/acns/hubble/events.log
    Read_from_Head      true
    DB                  /var/opt/microsoft/docker-cimprov/state/network_flow.db
    DB.Sync             Off
    Parser              json
    Rotate_Wait         20
    Refresh_Interval    5
    #${AZMON_STORAGE_TYPE}
    Skip_Long_Lines     On
    Ignore_Older        2m # configurable
   #${AZMON_TAIL_THREADED}
    ${TAIL_IGNORE_OLDER}

Reply:
added Threaded On